app/views/idv/doc_auth/document_capture.html.erb

```diff
@@ -17,57 +17,79 @@
   method: 'PUT',
   html: { autocomplete: 'off', role: 'form', class: 'mt2' }
 ) do |f| %>
   <%# ---- Front Image ----- %>
-  <%= render 'idv/doc_auth/front_of_state_id_image' %>
-
-  <h1 class='h3 my0'>
-    <%= t('doc_auth.headings.upload_front') %>
-  </h1>
-
-  <%= accordion('totp-info', t('doc_auth.tips.title_html'),
-    wrapper_css: 'my2 col-12 fs-16p') do %>
-    <%= render 'idv/doc_auth/tips_and_sample' %>
-    <div class='center'>
-      <%= image_tag(asset_url('state-id-sample-front.jpg'), height: 338, width: 450) %>
-    </div>
-  <% end %>
-
-  <%= f.input :front_image_data_url, as: :hidden %>
-  <%= render 'idv/doc_auth/notices', flow_session: flow_session %>
-  <%= f.input :front_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
-  <div class='my2' id='front_target'></div>
 
+  <hr class="mt3 mb3"/>
+  <div class="front-image">
+    <h2>
+      <%= t('doc_auth.headings.upload_front_html') %>
+    </h2>
+
+    <%= accordion('totp-info', t('doc_auth.tips.title_html'),
+      wrapper_css: 'my2 col-12 fs-16p') do %>
+      <%= render 'idv/doc_auth/tips_and_sample' %>
+      <div class='center'>
+        <%= image_tag(asset_url('state-id-sample-front.jpg'), height: 338, width: 450) %>
+      </div>
+    <% end %>
+
+    <%= f.input :front_image_data_url, as: :hidden %>
+    <%= f.input :front_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
+    <div class='my2' id='front_target'></div>
+  </div>
 
   <%# ---- Back Image ----- %>
-  <%= render 'idv/doc_auth/back_of_state_id_image' %>
 
-  <h1 class='h3 my0'>
-    <%= t('doc_auth.headings.upload_back') %>
-  </h1>
+  <hr class="mt3 mb3"/>
+  <div class="back-image">
+    <h2>
+      <%= t('doc_auth.headings.upload_back_html') %>
+    </h2>
+
+    <%= accordion('totp-info', t('doc_auth.tips.title_html'),
+      wrapper_css: 'my2 col-12 fs-16p') do %>
+      <%= render 'idv/doc_auth/tips_and_sample' %>
+      <div class='center'>
+        <%= image_tag(asset_url('state-id-sample-back.jpg'), height: 338, width: 450) %>
+      </div>
+    <% end %>
+
+    <%= f.input :back_image_data_url, as: :hidden %>
+    <%= f.input :back_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
+    <div class='my2' id='back_target'></div>
+  </div>
 
-  <%= accordion('totp-info', t('doc_auth.tips.title_html'),
-    wrapper_css: 'my2 col-12 fs-16p') do %>
-    <%= render 'idv/doc_auth/tips_and_sample' %>
-    <div class='center'>
-      <%= image_tag(asset_url('state-id-sample-back.jpg'), height: 338, width: 450) %>
+  <%# ---- Selfie ----- %>
+  <% if liveness_checking_enabled? %>
+    <hr class="mt3 mb3"/>
+    <div class="selfie">
+      <h2>
+        <%= t('doc_auth.headings.selfie') %>
+      </h2>
+
+      <%= f.input :selfie_image_data_url, as: :hidden %>
+      <%= f.input :selfie_image, label: false, as: :file, required: true, wrapper_class: 'mt3 sm-col-8' %>
+      <div class='my2' id='selfie_target'>
```

Comment:
Just noting that this tag is unclosed.

```suggestion
      <div class='my2' id='selfie_target'></div>
```

I encountered it in #3986. Not sure it's the cause of the errors there, but it was the source of some confusion in trying to keep the static rendered "Start over" and "Cancel" buttons, and them not showing 😅  I'll plan to include a fix for it with those changes.